llvm/include/llvm/Support/BlockFrequency.h

```diff
@@ -16,6 +16,8 @@
 #include <cassert>
 #include <cstdint>
 
+#include "llvm/Support/Compiler.h"
```

Comment:
Once we switch to `[[nodiscard]]`, we shouldn't have to include `Compiler.h` here.